pkg/backingstore/reconciler.go

```diff
@@ -795,3 +837,116 @@ func (r *Reconciler) ReconcilePool() error {
 
 	return nil
 }
+
+func (r *Reconciler) reconcilePvPool() error {
```

Comment:
several issues I see with `reconcilePvPool`:

1. why isn't scale down supported? it was supported before
2. the naming convention for the pods and pvc can cause problems once we will support scale down.   
when you use sequential names (backingstore-1, backingstore-2, etc.) and you need to scale down then you are forced to remove the last one so you can keep the naming order. then if the user wants to scale back up before the previous scale down ended, it's becoming complicated to keep track of the state of previous scale down

Reply:
changing naming convention - scale down only to the number of already connected hosts in noobaa